plugins/optimization-detective/tests/storage/test-rest-api.php

```diff
@@ -349,16 +349,14 @@ public function data_provider_invalid_params(): array {
 				'params'          => array_merge(
 					$valid_params,
 					array(
-						// Repeat the elements until the JSON will surpass 64 KiB.
-						'elements' => array_fill(
-							0,
-							200,
+						// Fill the JSON with more than 64KB of data.
+						'elements' => array(
 							array_merge(
 								$valid_element,
 								array(
-									'xpath' => '/HTML/BODY/DIV[@id=\'page\']/*[1][self::DIV]',
+									'xpath' => bin2hex( random_bytes( 65000 ) ),
```

Comment:
So that the XPath still matches the required pattern, even if we never get to the point where the pattern is checked in our test:

```suggestion
									'xpath' => sprintf( '/HTML/BODY/DIV[@id=\'%s\']/*[1][self::DIV]', bin2hex( random_bytes( 65000 ) ) ),
```